### DISCUSSION TITLE
handbook/integration/visual-studio-code/

### DISCUSSION THREAD

**Discussion by giscus** [serenity-js/serenity-js#1855] [Comments]
# handbook/integration/visual-studio-code/

Serenity/JS integrates with industry-standard test runners, so that your test scenarios can be executed

https://serenity-js.org/handbook/integration/visual-studio-code/

**Comment by VasuQantas23** [comment]
Hi mate, Looking forward to see this article published soon :)

**Reply by jan-molak** [reply]
OK, so it's taken _a bit_ longer than I though, but here it is :-) Thoughts and feedback welcome!

**Comment by Fcoder56** [comment]
hello
